utils/tests/testRunner.sh

```diff
@@ -35,9 +42,10 @@ done
 
 # Install @ima scoped packages from local registry
 npm config set @ima:registry=$NPM_LOCAL_REGISTRY_URL
+npm config set legacy-peer-deps true
```

Comment:
Why are we using this? We should not need legacy-peer-deps anywhere.

Reply:
Removed.